src/Toml/Printer.hs

```diff
@@ -136,7 +141,7 @@ prettyKeyValue options i =
 -- | Returns pretty formatted tables section of the 'TOML'.
 prettyTables :: PrintOptions -> Int -> Text -> PrefixMap TOML -> Text
 prettyTables options i pref =
-    Text.concat . map (prettyTable . snd) . orderWith options . HashMap.toList
+    Text.intercalate "\n" . map (prettyTable . snd) . orderWith options . HashMap.toList
```

Comment:
This also looks strange. Do you have any idea why `Text.concat` previously was enough and now this is `intercalate`?